integrations/http_bin_org.go

```diff
@@ -0,0 +1,65 @@
+package integrations
+
+import (
+	"github.com/buger/jsonparser"
+	"io/ioutil"
+	"net"
+	"net/http"
+)
+
+const HttpBinExtIPEndpoint = "http://httpbin.org/ip"
+
+type HttpBin struct {
```

Comment:
Codacy found an issue: [type HttpBin should be HTTPBin](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)